src/Controller/TreeController.php

```diff
@@ -102,6 +102,10 @@ public function reorderAction(Request $request)
             return new JsonResponse(array('Parameters parent, dropped and target has to be set to reorder.'), Response::HTTP_BAD_REQUEST);
         }
 
+        if (in_array($position, array('over', 'child'))) {
+            return new JsonResponse(array('Can not resort into a collection.'), Response::HTTP_BAD_REQUEST);
```

Comment:
now i see your issue, i am not that good in wording, even in my home language :-) So what do i mean, based on the description before?